crates/context/config/src/client/env/proxy/mutate.rs

```diff
@@ -1,8 +1,86 @@
-use crate::client::CallClient;
+use std::marker::PhantomData;
 
-mod propose;
+use ed25519_dalek::{Signer, SigningKey};
+
+use crate::client::env::{utils, Method};
+use crate::client::protocol::near::Near;
+use crate::client::protocol::starknet::Starknet;
+use crate::client::transport::Transport;
+use crate::client::{CallClient, ClientError, Operation};
+use crate::types::Signed;
+use crate::{ProposalWithApprovals, ProxyMutateRequest};
+
+pub mod methods;
 
 #[derive(Debug)]
 pub struct ContextProxyMutate<'a, T> {
+    pub client: CallClient<'a, T>,
+}
+
+#[derive(Debug)]
+pub struct ContextProxyMutateRequest<'a, T> {
     client: CallClient<'a, T>,
+    raw_request: ProxyMutateRequest,
+}
+
+#[derive(Debug)]
+struct Mutate<'a> {
+    pub(crate) signing_key: [u8; 32],
+    pub(crate) raw_request: ProxyMutateRequest,
+    _marker: PhantomData<&'a ()>,
```

Comment:
@miraclx I'll need some guidance here. Not sure what is the best way to handle the lifetime

Reply:
you don't need one here

Reply:
Removed